`python312Packages.moviepy` build failure on x86_64-linux as of `48134064`

## Issue description
Build of `python312Packages.moviepy` failed on x86_64-linux as of 48134064dd7b3f2224a05a9ee5e206680c9d93bc. This is currently breaking `python3Packages.wandb`.

```
error: builder for '/nix/store/049yrmcxrd0svaqycablj7f56wbahzp6-python3.12-moviepy-2.1.2.drv' failed with exit code 1;
       last 10 log lines:
       >   File "/nix/store/0l539chjmcq5kdd43j6dgdjky4sjl7hl-python3-3.12.8/lib/python3.12/threading.py", line 655, in wait
       >     signaled = self._cond.wait(timeout)
       >   File "/nix/store/0l539chjmcq5kdd43j6dgdjky4sjl7hl-python3-3.12.8/lib/python3.12/threading.py", line 359, in wait
       >     gotit = waiter.acquire(True, timeout)
       > +++++++++++++++++++++++++++++++++++ Timeout ++++++++++++++++++++++++++++++++++++
       > ----------------------------- Captured stderr call -----------------------------
       > frame_index:  98%|█████████▊| 515/528 [00:29<00:00, 16.23it/s, now=None]
       > =========================== short test summary info ============================
       > FAILED tests/test_issues.py::test_issue_1682 - Failed: Timeout >30.0s
       > ================ 1 failed, 200 passed, 45 deselected in 58.94s =================
       For full logs, run 'nix log /nix/store/049yrmcxrd0svaqycablj7f56wbahzp6-python3.12-moviepy-2.1.2.drv'.
error: 1 dependencies of derivation '/nix/store/liag4v3l6fyfblpari966p0zhnpmy6ww-python3.12-wandb-0.19.6.drv' failed to build
```

[full build log](https://github.com/samuela/nixpkgs-upkeep/actions/runs/13319618548)

python3Packages.wandb maintainers cc: @samuela
python312Packages.moviepy maintainers cc: 
Other cc: n/a

### Steps to reproduce
1. Checkout nixpkgs at commit 48134064dd7b3f2224a05a9ee5e206680c9d93bc
2. Run `nix-build -A python312Packages.moviepy`

## Technical details
```
 - system: `"x86_64-linux"`
 - host os: `Linux 6.8.0-1021-azure, Ubuntu, 24.04.1 LTS (Noble Numbat), nobuild`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.16.1`
 - nixpkgs: `/nix/store/pllfpwwwnhjjbvv6df9qpna33nrm78qs-source`
```

Contents of `~/.config/nixpkgs/config.nix`:
```
{ allowUnfree = true; cudaSupport = true; cudaCapabilities = ["8.0"]; }
```

## Misc.
This issue was automatically generated by [nixpkgs-upkeep](https://github.com/samuela/nixpkgs-upkeep).
- [CI workflow](https://github.com/samuela/nixpkgs-upkeep/actions/runs/13319618548) that created this issue.
- Internal tags: -4863022444284443904 -8586873226307209948

--- a/pkgs/development/python-modules/moviepy/default.nix
+++ b/pkgs/development/python-modules/moviepy/default.nix
@@ -71,7 +71,8 @@ buildPythonPackage rec {
     pytestCheckHook
   ] ++ lib.flatten (lib.attrValues optional-dependencies);
 
-  pytestFlagsArray = [ "--timeout=30" ];
+  # See https://github.com/NixOS/nixpkgs/issues/381908.
+  pytestFlagsArray = [ "--timeout=60" ];
 
   pythonImportsCheck = [ "moviepy" ];
 

